workers-scale.go

```diff
@@ -118,7 +121,10 @@ func NewWorkersScale(metricsEndpoint *string, ocpMetaAgent *ocpmetadata.Metadata
 				ConfigSpec:      &workloads.ConfigSpec,
 				MetricsEndpoint: *metricsEndpoint,
 				UserMetaData:    userMetadata,
-				MetricsMetadata: metadata,
+				MetricsMetadata: map[string]interface{}{
```

Comment:
Keeping only limited amount of metadata in metrics to be inline with other metrics in wrapper.